### PR TITLE
PLATOPS-1621 build.sbt as recommended per warning, and crypto upgrade.

### PR DESCRIPTION
**Pull request by jm2dev**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "uk.gov.hmrc"       %% "crypto"      % "5.1.0",
+  "org.scalatest"     %% "scalatest"   % "3.0.1" % "test",
+  "org.pegdown"       %  "pegdown"     % "1.6.0" % "test"
+)
+
+lazy val library = Project(appName, file("."))
+  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+  .settings(
+    libraryDependencies ++= dependencies,
+    organization := "uk.gov.hmrc",
```
Comment (konradwudkowski) [inline build.sbt]: organization is already set by `SbtAutoBuildPlugin`

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import sbt.Keys._
+import sbt._
+import uk.gov.hmrc.versioning.SbtGitVersioning
+
+val appName = "json-encryption"
+
+val dependencies = Seq(
+  "com.typesafe.play" %% "play-json"   % "2.6.5" % "provided",
```
Comment (konradwudkowski) [inline build.sbt]: I'm a bit worried by this play 2.6.5 even though it has nothing to do with upgrading crypto

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,8 +1,10 @@
 resolvers += Resolver.url("hmrc-sbt-plugin-releases",
   url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += Resolver.typesafeRepo("releases")
```
Comment (konradwudkowski) [inline project/plugins.sbt:4]: also requires the following resolver:

```
resolvers += Resolver.bintrayRepo("hmrc", "releases")
```

File: build.sbt (hunk 1 above)
Comment (jm2dev) [reply build.sbt]: Updated.

File: project/plugins.sbt (hunk 3 above)
Comment (jm2dev) [reply project/plugins.sbt:4]: May I ask why?

Comment (jm2dev) [reply project/plugins.sbt:4]: Added hmrc releases resolver.

File: build.sbt (hunk 2 above)
Comment (jm2dev) [reply build.sbt]: Changed to from hardcoded 2.6.5 to PlayVersion.current.

File: project/plugins.sbt [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
+resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "0.9.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.13.0")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.15.0")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "0.14.0")
+
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
```
Comment (konradwudkowski) [inline project/plugins.sbt]: We are bringing entire play as a dependency + the plugin but we only need `play-json`

File: build.sbt [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import sbt.Keys._
+import sbt._
+import uk.gov.hmrc.versioning.SbtGitVersioning
+import play.sbt.PlayImport.filters
+import play.core.PlayVersion
+
+val appName = "json-encryption"
+
+val dependencies = Seq(
+  "com.typesafe.play" %% "play-json" % PlayVersion.current % "provided",
```
Comment (konradwudkowski) [inline build.sbt]: "2.5.12" would be lighter than `PlayVersion.current`

Comment (jm2dev) [reply build.sbt]: Ok, then I'll revert that change. It seems it doesn't provide the results I expected.

**Review by konradwudkowski — APPROVED** [review]
LGTM

**Pull request merged**